pythoncalculator/divide.py

```diff
@@ -0,0 +1,2 @@
+def divide(x, z):
+    return x / y
```

Comment:
Typo should perhaps be `z`....
```suggestion
    return x / z
```